README.adoc

```diff
@@ -15,27 +15,27 @@
 :page-related-guides: ['microprofile-config', 'microprofile-metrics', 'cdi-intro', 'rest-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
 :source-highlighter: prettify
-= Adding health reports to a microservice
+= Adding health reports to microservices
 
-You'll explore how to report and check the health of a microservice with MicroProfile Health.
+Explore how to report and check the health of a microservice with MicroProfile Health.
```

Comment:
I thought "You'll explore" was ID edit, but now looks like we have them both in different guides. Not a big deal, thought just to mention it.

Reply:
oh, the two guides I wrote had "Explore". I think this is fine

Reply:
Sure, np.